builds/fabric8-che/assembly/dashboard/src/main/patches/src/app/index.module.ts.patch

```diff
@@ -105,7 +110,7 @@
  
  // add interceptors
  initModule.factory('ETagInterceptor', ($window, $cookies, $q) => {
-@@ -361,6 +445,7 @@
+@@ -361,6 +450,7 @@
```

Comment:
@gorkem Do you think we could leave this for a future PR ? in order to avoid adding changes that are not related to the PR main purpose ?